package.json

```diff
@@ -32,6 +32,7 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/react": "^18.0.9",
```

Comment:
Lets remove this from here too we already have `@types/react` in nextjs

You could run `yarn remove @types/react` and it should remove it from root

Reply:
Forgot to remove that one, sorry!! I updated the file and removed that line.